docs/configuration/apis.md

```diff
@@ -14,7 +14,7 @@ If per-API and root settings modify the same properties, per-API settings overri
 ## Patterned properties
 
 {% json-schema
-  schemaFilePath="./api.yaml"
+  schema="./api.yaml"
```

Comment:
This is not a valid format.

This is the correct one:

```suggestion
  schema={"$ref" : "./api.yaml"}
```

Reply:
Oops. Thank you!